Move solr config to solr/config

aelkiss opened the pull request:
Puts the solr configuration where we expect it to be for deployment.

billdueber commented:
Merge pending changing everything else that expects the solr config to be in a particular spot.

billdueber commented:
@aelkiss There are plans to possibly extend dromedary to use multiple cores, and this style of configuration doesn't seem like it could support that. Not a current, urgent need, but something to be aware of. Also gonna be true for, e.g., HT Collection Builder.

aelkiss commented:
If it's easier we can make deployment configurable with respect to where to find the solr config.

aelkiss commented:
We would still need to change deployment to understand multiple solr cores, but that wouldn't be a requirement now I guess.

malakai97 commented:
It might be easiest to just add and commit a symlink to the new location from the old one.

aelkiss commented:
OK, I reverted the move and changed it just to be a symlink. Deployment uses some wacky git -> svn magic, so I'm not sure this will work; let me double check before you merge this.

aelkiss commented:
Yeah, the symlink doesn't work with the svn wackiness. I'll just support alternate paths in the deployment.
